README.md

```
@@ -1 +1,6 @@
-# ft_irc
\ No newline at end of file
+# ft_irc
+
+while (newCapacity < neededSize)
+	newCapacity *= 2;
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)